app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -647,6 +647,13 @@ function useGrantDetail() {
     form.value.twitter = cleanTwitterUrl(grantMetadata.value?.properties?.twitterURI) || '';
   }
 
+  /**
+   * @notice auto open if URL has query edits
+   */
+  if (route.query && route.query.edit && isOwner) {
+    enableEdit();
+  }
+
```

Comment:
Should this be within an `onMounted` hook? It looks like it just lives directly in the composition function, so I wonder if there could be a race condition where this doesn't always execute properly if it's ran before mounting finishes? I'm not sure how it works when directly within the composition function like this

Reply:
moved it to onMount